src/telegram.js

```diff
@@ -845,6 +845,160 @@ class TelegramBot extends EventEmitter {
   }
 
   /**
+   * Use this method to restrict a user in a supergroup.
+   * The bot must be an administrator in the supergroup for this to work
+   * and must have the appropriate admin rights. Pass True for all boolean parameters
+   * to lift restrictions from a user. Returns True on success.
+   *
+   * @param {Number|String} chatId Unique identifier for the target chat or username of the target supergroup
+   * @param {String} userId
+   * @param {Object} [options] Additional Telegram query options
+   * @returns {Promise}
```

Comment:
For consistency, let's use `@return` not `@returns`.

*PS: Apply this change across the entire PR*